code/modules/custom_ka/projectiles.dm

```diff
@@ -7,38 +7,54 @@
 	check_armour = "bomb"
 	range = 5
 	var/pressure_decrease = 0.25
+	var/damage_s
```

Comment:
Please add comments to damage_s and aoe_s as their purpose is not evident from the name